style.css

```diff
@@ -272,11 +293,91 @@ i {
   background-color: #555;
 }
 
-#Projects button {
+button {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-26 23-47-31](https://user-images.githubusercontent.com/52934128/165405182-05da2572-b77c-4e42-88a6-9f950471af8d.png)

the button on the pressed state has the `background: #5E42D7;``

### EXAMPLE:
```
button:active {
   background-color: #5E42D7;
}
```
```
button:disabled {
    background-color: #2A2B33;
}
```